client/src/components/Home/index.tsx

```diff
@@ -391,6 +407,8 @@ export default class Home extends React.Component<any, IHomeState> {
             defaultValue={template && template.id || ''}
             lineDirection="center"
             placeholder="Choose an ID for the dashboard (will be used in the url)"
+            error={error}
```

Comment:
This is a little too specific - It means that even if there's an error not associated to this field or the creation process in general it would be assigned to this field (even if it's not visible).
I think a better approach would be to use Toast messages from the ConfigurationActions class - and make sure it comes on top of the active dialog.